### PR TITLE
Add option stopPropagation for swipe

### LINKED ISSUE #680: 组件swipe与tab-container一起使用出现的bug
mint-ui的`swipe`和`tab-container`一起用的时候，tab-container的`swipeable=true`的时候，滑动切换swipe内容时，会触发tab-container的切换事件,不仅仅swipe内容切换了，tab-container也切换了。
建议传递一个阻止冒泡的参数，当滑动`swipe`时，阻止`tab-container`的切换事件

示意图（gif图传不上去怎么回事？）

### PR DESCRIPTION
**Pull request by galenyuan**
fixed #680 

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow the contributing guide.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/swipe/src/swipe.vue [hunk 1]
```diff
@@ -119,6 +119,11 @@
       prevent: {
         type: Boolean,
         default: false
+      },
+
+      stopPropagation: {
```
Comment (QingWei-Li) [inline packages/swipe/src/swipe.vue]: 是不是太长了？

Comment (galenyuan) [inline packages/swipe/src/swipe.vue]: 想叫stop来着，但太歧义了，尤其是在一个有auto-play的组件里

Comment (QingWei-Li) [inline packages/swipe/src/swipe.vue]: 好的大佬

**Comment by zhao-ping** [comment]
现在可以使用更新过的swipe吗？我重新用NPM下载下来mint-ui，看了下swipe里的文件，里面并没有加上上面的那些处理bug的东西

**Comment by galenyuan** [comment]
@zhao-ping 晚点会发布版本

**Pull request merged**